types/types.go

```diff
@@ -36,12 +36,24 @@ type NetConf struct {
 	CNIDir  string `json:"cniDir"`
 	BinDir  string `json:"binDir"`
 	// RawDelegates is private to the NetConf class; use Delegates instead
-	RawDelegates []map[string]interface{} `json:"delegates"`
-	Delegates    []*DelegateNetConf       `json:"-"`
-	NetStatus    []*NetworkStatus         `json:"-"`
-	Kubeconfig   string                   `json:"kubeconfig"`
-	LogFile      string                   `json:"logFile"`
-	LogLevel     string                   `json:"logLevel"`
+	RawDelegates  []map[string]interface{} `json:"delegates"`
+	Delegates     []*DelegateNetConf       `json:"-"`
+	NetStatus     []*NetworkStatus         `json:"-"`
+	Kubeconfig    string                   `json:"kubeconfig"`
+	LogFile       string                   `json:"logFile"`
+	LogLevel      string                   `json:"logLevel"`
+	RuntimeConfig *RuntimeConfig           `json:"runtimeConfig,omitempty"`
```

Comment:
Small nitpick: RuntimeConfig is refer to https://github.com/containernetworking/cni/blob/d974c1dd54b1b15e2514b90f22ee68ab9740e442/libcni/api.go#L34  by definition, A RuntimeConf holds the arguments to one invocation of a CNI plugin excepting the network configuration, Is possible to change it to PortMapConfig ? @s1061123 @dougbtv. Please help me here, if I am missing something.

Reply:
I was seeing it as a placeholder for the runtimeConf attribute of the JSON given in the stdin of multus. I think it will probably be populated by other runtime configs in the future (ingress, egress, bandwidth), am I wrong? In this case, it won't be only related to the portMap config.

However, I agree the naming overlap is confusing, may be we should rename it.

Reply:
That is a cool approach `I think it will probably be populated by other runtime configs in the future (ingress, egress, bandwidth), am I wrong?` I am convinced on this use case.